### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by matthewsearle01**

### CODE REVIEW

File: css/dark-theme.css [hunk 1]
```diff
@@ -187,39 +192,49 @@ a:hover {
...
     border-radius: 75% 25% 43% 57% / 67% 45% 55% 33%;
+    color: #18112E;
 }
 
 .project-card {
-    width: 40rem;
-    min-height: 50rem;
+    background-color: #F5F5F5;
+    border-radius: 50px;
     border: 1px solid black;
```
Comment (nealioa) [inline css/dark-theme.css:203]: Named color. Use the hash code.

File: css/light-theme.css [hunk 2]
```diff
@@ -191,38 +196,48 @@ a:hover {
...
     border-radius: 75% 25% 43% 57% / 67% 45% 55% 33%;
+    color: #18112E;
 }
 
 .project-card {
-    width: 40rem;
-    min-height: 50rem;
+    background-color: #E2E8F0;
+    border-radius: 50px;
     border: 1px solid black;
```
Comment (nealioa) [inline css/light-theme.css:206]: Named colour again.

File: index.html [hunk 3]
```diff
@@ -13,55 +13,59 @@
...
-            </button>
-        </div>
-
-                <div class="headshot-container">
-                    <img class="headshot" src="images/IMG_0137.jpg" alt="matt searle headshot">
-                </div>
-
-                <!--Introduction Text-->
-                <h2>Matt Searle</h2>
+                <!--Main Section of Landing Page-->
```
Comment (nealioa) [inline index.html:16]: Is this comment meant to be indented this far?

File: index.html [hunk 4]
```diff
@@ -13,55 +13,59 @@
...
+                    I was drawn to software development out of a desire to work in an exciting innovative industry, and I am currently looking for a role with room for creativity and collaboration that will challenge me to constantly learn and grow.
+                </p>
+                <p class="content">
+                    After studying music technology at university I moved into the music industry, first as a technical assistant in radio, then at various record shops until ultimately finding myself as a specialist music buyer for one of the world's largest record stores.
                 </p>
+                <p class="content">
+                    But I have had a strong and ever increasing desire to get back into using technology to create, and after discovering coding I was hooked from my very first "Hello, world!". The excitement about learning new technologies, the 
+                    challenge to solve a tricky problem, and the satisfaction of seeing my code realised as a working application, all led me to go all in and enrol on the full-stack coding bootcamp at iO Academy.
+                </p>
+
```
Comment (nealioa) [inline index.html:62]: Is this blank line necessary?

**Review by nealioa — CHANGES_REQUESTED** [review]

File: css/light-theme.css (hunk 2 above)
Comment (matthewsearle01) [reply css/light-theme.css]: thanks, fixed

File: index.html (hunk 3 above)
Comment (matthewsearle01) [reply index.html]: thanks, fixed

File: index.html (hunk 4 above)
Comment (matthewsearle01) [reply index.html]: thanks, deleted this line

File: css/dark-theme.css (hunk 1 above)
Comment (matthewsearle01) [reply css/dark-theme.css]: thanks, fixed

**Review by nealioa — APPROVED** [review]

**Pull request merged**